dev/gen-anvil-state.sh

```
@@ -0,0 +1,80 @@
+#
+# This script is intended to be run inside a docker container.
+# It will start an anvil instance, deploy the contracts, and dump the state info.
+#
+
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)